backend/oasst_backend/user_repository.py

```diff
@@ -207,6 +207,7 @@ def query_users_ordered_by_display_name(
         limit: Optional[int] = 100,
         desc: bool = False,
     ) -> list[User]:
+
```

Comment:
this newline triggers my review as code owner. it's a very beautiful newline. I approve ;)